packages/slice-machine/components/Preview/components/Header/index.tsx

```diff
@@ -23,7 +23,7 @@ const redirect = (
   isPreview?: boolean
 ): void => {
   if (!variation) {
-    router.push(`/${model.href}/${model.infos.sliceName}`);
+    void router.push(`/${model.href}/${model.infos.sliceName}`);
```

Comment:
i think `router.push` is async, which might explain the need for void

Reply:
yes that's why :)